indexer/run.sh

```diff
@@ -33,23 +33,96 @@ fetch --nohooks chromium
 
 cd src
 
+mkdir -p out/Build
+export BUILD_DIR=$(readlink -f out/Build)
```

Comment:
i think we should use `out/Default`. even though it is not a principled solution it should at least enable users with default build directory to have more features working for generated symbols as well.